transmart-core-db/src/main/groovy/org/transmartproject/db/tree/TreeNode.groovy

```diff
@@ -268,4 +271,20 @@ class TreeNode {
         }
     }
 
+    final String getDimension() {
+        switch (tableName) {
+            case 'concept_dimension':
+                return DimensionImpl.CONCEPT.name
+            case 'patient_dimension':
+                return DimensionImpl.PATIENT.name
+            case 'modifier_dimension':
+                return 'modifier'
```

Comment:
but modifier_code is filled only for sample_type dimension, so also our test data should be updated then